General Definition Cleanup - 10/20/2024

robertgk2017 opened the pull request:
TES3 - Disable HITME check
TES3 - Misc organizational cleanup

TES4 \ wbConditionFunctions - Add 3 missing OBSE Functions
TES4 \ CELL - Move XCLC to end of the definition to fix TwbMainRecord.GetDisplayName
TES4 \ CELL - Update AfterLoad/DontShows
TES4 \ PACK \ PTDT \ Target \ Object Type - Define Enum
TES4 \ PGRD/ROAD - Update Various Sub-Records

TES4 - Cleanup/Organize Conditions Functions
TES4 - Collapse wbEffects to a single Variable
TES4 - Convert AfterLoads to .SetAfterLoad
TES4 - Convert aRequireds to .SetRequired
TES4 - Convert DontShows to .SetDontShow
TES4 - Formatting Cleanup
TES4 - Organize/Re-order all of the functions that precede the DefineTES4 Procedure
TES4 - Organize/Re-order the DefineTES4 Procedure
TES4 - Sort wbFormIDCk aValidSigs
TES4 - Update Required status and Default Values across all records
TES4 - Update wbActorValueEnum
TES4 - Update wbCTDA, renamed to wbConditions
TES4 - Update wbEnchantment handling
TES4 - Update wbFormTypeEnum to only contain valid values
TES4 - wbEnum and wbFlags consistency

TES4/FO3/FNV - combine wbSoundTypeSoundsOld with the one in DefsCommon

FO3 \ wbConditions - add wbFromVersion checks

FO3 - Cleanup/Organize Conditions Handling
FO3 - Cleanup wbEnums
FO3 - Condition Functions formatting cleanup
FO3 - Convert AfterLoads, AfterSets, DontShows, Requireds to .Set Callbacks
FO3 - Organize uses/variables
FO3 - Sort, Organize, Condense Variables
FO3 - Update wbFormTypeEnum

FO3/FNV \ HDPT \ DATA - Fix Typo for Reachability
FO3/FNV \ IDLE - Update ANAM
FO3/FNV \ NAVI - Decoded the rest of NVMI, Added Summaries/Sorting
FO3/FNV \ PACK - Cleanup Type Specific Flags

FNV - Update Conditions Handling

TES5 - Add wbDIALAddInfo
TES5 - Update Conditions Handling
TES5 \ DIAL \ BNAM - Make reachable
TES5 \ STAT \ DNAM - Simplify IsSSE check

TES4/FO4 \ FLOR \ PFIG - Updated accepted form ids

TES5/FO4/FO76/SF1 \ NAVI - Made more consistent across the games
TES5/FO4/FO76/SF1 \ SMBN/SMEN/SMQN - Misc Cleanup

FO4/FO76/SF1 \ FURN - Remove unused wbSkillEnum and adjusted WBDT
FO4/FO76/SF1 - Convert Gender enums to wbSexEnum

FO76 \ CELL/LGTM - Skybox Bounds decoded, confirmed by Eck

SF1 \ QUST - Remove redundant wbQuestEventTypeEnum

FO3/FNV/TES5/FO4/FO76/SF1 \ BPDT - Update BodyPart Array's SortKey and Summary.

ALL \ TES4 - Add/Update Flag 4 = Optimized
ALL \ CELL - Update wbCellGrid and add DontShow/IsRemovables to XCLC and XCLL
ALL \ GMST - Fill names + dfFlags
ALL \ IDLE \  DATA/ANAM - Update to reflect Parent/Previous IDLE Behavior
ALL \ NAVI \ NVMI \ Island Data - Add dfFlags
ALL \ REGN - Clean up RDAT Flags

ALL - Convert Unused ByteArrays to wbUnused
ALL - Convert XYZ Float Structs to wbVec3 where possible
ALL - Set default value for inventory/leveled list entry counts to 1
ALL - Update AddInfos to condense their code + Add new simple ones
ALL - Update wbLeveledListEntry handling
ALL - Update wbOwnership handling

Interface - Add nil default value to aSkipSigs in all instances of RStruct/RUnion

Script \ Clean Edge Links in Navmeshes - Fix not moving Edge Link Type

Added the following new Enums:
TES4 - wbAttributeEnum
TES4 - wbBipedFlags
TES4 - wbMajorSkillsEnum
TES4 - wbResistEnum
TES4 - wbSoundMembers

Moved the following shared defs to Common:
wbAliasToStr
wbBodyPartIndexEnum
wbClmtMoonsPhaseLength 
wbClmtTime 
wbCTDACompValueDecider - Renamed to wbConditionCompValueDecider
wbCtdaTypeAfterSet  - Renamed to wbConditionTypeAfterSet
wbCtdaTypeToInt - Renamed to wbConditionTypeToInt
wbCTDAType/wbCTDATypeToStr - Renamed to wbConditionTypeToStr
wbCOEDOwnerDecider
wbCreatureTypeEnum 
wbCriticalStageEnum 
wbCrimeTypeEnum 
wbEffectTypeEnum 
wbEquipTypeEnum 
wbGMSTUnionDecider 
wbMenuModeEnum
wbMoodEnum 
wbMoralityEnum 
wbPlacedAddInfo 
wbPKDTFlags - Renamed to wbPackageFlags
wbPKDTType - Renamed to wbPackageTypeEnum
wbRagdoll 
wbRegionAreas 
wbRPLDAfterLoad 
wbServiceFlags 
wbSoulGemEnum
wbTemplateFlags 
wbTexturedModel

Removed the following unused/useless defs:
wbIPDSDATACount
wbKeywordsAfterLoad
wbPKDTSpecificFlagsUnused
wbRemoveRNAM

robertgk2017 committed:
ALL - Convert XYZ Float Structs to wbVec3 where possible

robertgk2017 committed:
Move wbSoulGemEnum to Common

robertgk2017 committed:
Move wbCOEDOwnerDecider to Common

robertgk2017 committed:
Script \ Clean Edge Links in Navmeshes - Fix not moving Edge Link Type

robertgk2017 committed:
TES5/FO4/FO76/SF1 \ NAVI \ NVMI \ Island Data - Add dfFlags

robertgk2017 committed:
ALL \ REGN - Clean up RDAT Flags

robertgk2017 committed:
Move wbRPLDAfterLoad to Common

robertgk2017 committed:
Move wbPlacedAddInfo to common

robertgk2017 committed:
Update wbCellAddInfo

robertgk2017 committed:
Update wbNAVMAddInfo

robertgk2017 committed:
Typo

robertgk2017 committed:
Update all AddInfos to condense their code + Add new simple ones

robertgk2017 committed:
Forgot TES5

robertgk2017 committed:
Move wbClmtTime to Common

robertgk2017 committed:
Move wbClmtMoonsPhaseLength to Common + Fix Masser/Secunda Bits

robertgk2017 committed:
Remove unused/unneeded wbGLOBFNAM function

robertgk2017 committed:
wbPlacedAddInfo - Account for Deleted Refs

robertgk2017 committed:
Move wbGMSTUnionDecider to Common

robertgk2017 committed:
ALL \ GMST - Fill names + dfFlags

robertgk2017 committed:
.SetRequired doesn't need to declare True :)

robertgk2017 committed:
And more.

robertgk2017 committed:
wbGMSTUnionDecider - Enable U in SF

robertgk2017 committed:
SF1 \ GMST - Add Unsigned Int Value Type

robertgk2017 committed:
Typo

robertgk2017 committed:
ALL \ LIGH - Remove unneeded AftrerLoad and set the defaults it applied directly.

robertgk2017 committed:
TES4/FNV/TES5/FO4/SF1 - Update GRUP write orders

robertgk2017 committed:
Delete extra newline

robertgk2017 committed:
move wbXRGD and wbXRGB to Common via a single wbRagdoll

robertgk2017 committed:
SF - Disable force loading Main Blueprints

robertgk2017 committed:
TES5/FO4/FO76/SF1 \ SMBN/SMEN/SMQN - Misc Cleanup

robertgk2017 committed:
Move wbCreatureTypeEnum to Common

robertgk2017 committed:
ALL - Update wbOwnership handling

robertgk2017 committed:
Add nil default value to aSkipSigs in RStruct/RUnion

robertgk2017 committed:
SF1 \ QUST - Remove redundant wbQuestEventTypeEnum

2nd enum for the same thing that already existed?

robertgk2017 committed:
FO4/FO76/SF1 - Convert Gender enums to wbSexEnum

robertgk2017 committed:
Merge branch 'dev-4.1.6' of https://github.com/TES5Edit/TES5Edit into Misc

robertgk2017 committed:
Resolve Conflicts

robertgk2017 committed:
Update ESL Script

robertgk2017 committed:
Update ESL Script Again

robertgk2017 committed:
TES3 - convert NiPoint3's to wbVec3

robertgk2017 committed:
TES3 - Disable HITME check

robertgk2017 committed:
Move wbTexturedModel to Common

robertgk2017 committed:
TES4 - Convert Unused ByteArrays to wbUnused

robertgk2017 committed:
Interface - Set wbUnused to wbNeverShow

robertgk2017 committed:
ALL - Convert Unused ByteArrays to wbUnused

robertgk2017 committed:
TES4 \ CELL - Move XCLC to end of the definition to fix TwbMainRecord.GetDisplayName

robertgk2017 committed:
ALL \ CELL - Update wbCellGrid and add DontShow/IsRemovables to XCLC and XCLL

robertgk2017 committed:
Remove Unused wbKeywordsAfterLoad

robertgk2017 committed:
Move wbCrimeTypeEnum to Common

robertgk2017 committed:
Move wbPKDTType to Common, rename to wbPackageTypeEnum

robertgk2017 committed:
Forgot to update the saves files

robertgk2017 committed:
and 2 more....

robertgk2017 committed:
Move wbPKDTFlags to Common, Rename to wbPackageFlags

robertgk2017 committed:
Remove useless wbPKDTSpecificFlagsUnused

robertgk2017 committed:
And more.

robertgk2017 committed:
FO3/FNV \ PACK - Cleanup Type Specific Flags

robertgk2017 committed:
TES5 \ AVIF - Add wbAVIFSkillAfterLoad to Skill Category

robertgk2017 committed:
Move wbServiceFlags to Common

robertgk2017 committed:
FO4/FO76/SF1 \ FURN - Remove unused wbSkillEnum and adjusted WBDT

robertgk2017 committed:
Move wbTemplateFlags to Common

robertgk2017 committed:
wbServiceFlags - set aUnknownIsUnused = True

robertgk2017 committed:
Move wbBodyPartIndexEnum to Common, remove from TES5+ as its unused in the newer games

robertgk2017 committed:
Move wbCriticalStageEnum to Common

robertgk2017 committed:
Cleanup FO4+ IfThens in Common

robertgk2017 committed:
Move wbEquipTypeEnum to Common

robertgk2017 committed:
Move wbMenuModeEnum to Common

robertgk2017 committed:
Move wbMoodEnum to Common

robertgk2017 committed:
Add wbMoralityEnum to common

robertgk2017 committed:
TES4 \ wbCTDAFunctions - Add 3 missing OBSE Functions

robertgk2017 committed:
Move wbCTDAType/wbCTDATypeToStr to Common, rename to wbConditionTypeToStr

robertgk2017 committed:
Move wbCtdaTypeToInt to Common, rename to wbConditionTypeToInt

robertgk2017 committed:
Move wbCtdaTypeAfterSet to Common, rename to wbConditionTypeAfterSet

robertgk2017 committed:
Move wbCTDACompValueDecider to Common, rename wbConditionCompValueDecider

robertgk2017 committed:
ALL - Set default value for inventory/leveled list entry counts to 1

robertgk2017 committed:
TES4 - Update wbCTDA, renamed to wbConditions

robertgk2017 committed:
TES4 - Organize/Re-order all of the functions that precede the DefineTES4 Procedure

robertgk2017 committed:
TES4 - Organize/Re-order the DefineTES4 Procedure

robertgk2017 committed:
TES4 - Fix Initialization Error

robertgk2017 committed:
TES4 - Update wbActorValueEnum

robertgk2017 committed:
TES4 - Add wbAttributeEnum

robertgk2017 committed:
TES4 - Add wbMajorSkillsEnum

robertgk2017 committed:
TES4 - Add wbResistEnum

robertgk2017 committed:
Replace instances of wbActorValueEnum with more specific enums where appropriate

robertgk2017 committed:
TES4 - Update wbFormTypeEnum to only contain valid values

robertgk2017 committed:
TES4 - Update wbSkillEnum and wbSpecializationEnum

robertgk2017 committed:
TES4 - Update wbHeadPartIndexEnum and wbMagicSchoolEnum

robertgk2017 committed:
FO76 \ CELL/LGTM - Skybox Bounds decoded, confirmed by Eck

robertgk2017 committed:
TES5 - Remove unused wbRemoveRNAM

robertgk2017 committed:
TES4 - Sort wbFormIDCk aValidSigs

robertgk2017 committed:
TES4 \ wbSkillEnum - Convert from Signed to Unsigned Integer, fixes initialization assertion

robertgk2017 committed:
Fix more errors

robertgk2017 committed:
TES4/FO3/FNV - combine wbSoundTypeSoundsOld with the one in DefsCommon

robertgk2017 committed:
Fix IfThen order

robertgk2017 committed:
TES4 - Update Uses

robertgk2017 committed:
TES4 - Update wbEnchantment handling

robertgk2017 committed:
TES4 - Move ALCH

robertgk2017 committed:
Common - misc fixups

robertgk2017 committed:
TES4 - Collapse wbEffects to a single Variable

robertgk2017 committed:
ALL - Update wbLeveledListEntry handling

robertgk2017 committed:
TES5 - Add wbDIALAddInfo

robertgk2017 committed:
oops

robertgk2017 committed:
TES4 - Convert DontShows to .SetDontShow

robertgk2017 committed:
TES4 - Convert AfterLoads to .SetAfterLoad

robertgk2017 committed:
TES4 - Convert aRequireds to .SetRequired

robertgk2017 committed:
TES4 - Formatting Cleanup

robertgk2017 committed:
TES4 - wbEnum and wbFlags consistency

robertgk2017 committed:
ALL - Convert AddInfos to .SetAddInfo

robertgk2017 committed:
TES4 - Variable cleanup

robertgk2017 committed:
TES4 - wbSoundMembers and wbBipedFlags

robertgk2017 committed:
ALL - Move wbRegionAreas to Common

robertgk2017 committed:
TES4 - Shrink single element wbArrays

robertgk2017 committed:
TES4 \ PGRD/ROAD - Update Various Sub-Records

robertgk2017 committed:
TES4 - Error Fixes

robertgk2017 committed:
TES4 - Update Required status and Default Values across all records

robertgk2017 committed:
FO3 - Convert AfterLoads, AfterSets, DontShows, Requireds to .Set Callbacks

robertgk2017 committed:
FO3 - Organize uses/variables

robertgk2017 committed:
TES4 \ ARMO/CLOT - Fix Error

robertgk2017 committed:
FO3 - Sort, Organize, Condense Variables

robertgk2017 committed:
ALL \ PACK \ wbPackageFlags - Set UnknownIsUnused to True

robertgk2017 committed:
Common \ wbClimateTiming - Fix Startup Error in SF1

robertgk2017 committed:
SF1 \ LVLP - Forgot Condtions in the common LVLO

jbostrus committed:
Merge remote-tracking branch 'remotes/origin/dev-4.1.6' into Misc

robertgk2017 committed:
wbINFOAddInfo - add check for games Pre-Skyrim

robertgk2017 committed:
TES4 \ CELL - Update AfterLoad/DontShows

robertgk2017 committed:
TES4 \ STAT - DMTL Decoded

robertgk2017 committed:
TES4 \ STAT \ DMTL - Better Name from Zilav

robertgk2017 committed:
SF1 \ Reference Records - Add/Update XLRL

robertgk2017 committed:
ALL \ TES4 - Add/Update Flag 4 = Optimized

robertgk2017 committed:
TES4/FO3/FNV \ EFIT \ wbEffectTypeEnum Move To Common

robertgk2017 committed:
DefsCommon - Formatting Cleanup

robertgk2017 committed:
TES5 \ STAT \ DNAM - Simplify IsSSE check

robertgk2017 committed:
TES4 - Add Numbered Comments to wbConditionParameters

robertgk2017 committed:
FO3 - Cleanup/Organize Conditions Handling

robertgk2017 committed:
TES4 - Cleanup/Organize Conditions Functions

robertgk2017 committed:
FO3 - Condition Functions formatting cleanup

robertgk2017 committed:
FO3 - Update wbFormTypeEnum

robertgk2017 committed:
FO3 - Cleanup wbEnums

robertgk2017 committed:
TES4/FO3 Variable Fix

robertgk2017 committed:
FO3 - Update wbFlags

C3pa commented on Core/wbDefinitionsFNV.pas:
```diff
@@ -7680,50 +7679,6 @@ procedure DefineFNV;
-          {18} 'Combat Controller',
-          {19} 'Package Type 19',
-          {20} 'Package Type 20',
-          {21} 'Alarm',
-          {22} 'Flee',
-          {23} 'Trespass',
-          {24} 'Spectator',
```
Looks like these were dropped in the new enumeration?

robertgk2017 replied:
That's because they didn't exist in the first place.  Those are not  valid package types in the CK.

robertgk2017 committed:
FO3 - More flags fixes

robertgk2017 committed:
TES4/FO3 - Int Fix

robertgk2017 committed:
FO3 - And More

robertgk2017 committed:
Oops

robertgk2017 committed:
Update wbBlendModeEnum and wbBlendOpEnum

robertgk2017 committed:
TES4 - Manually Verified every condition functions parameters

robertgk2017 committed:
TES4 - Fix Default Value FormIDs

robertgk2017 committed:
TES4 \ PACK \ PTDT \ Target \ Object Type - Define Enum

robertgk2017 committed:
FO3/FNV - Remove Unused wbIPDSDATACount

robertgk2017 committed:
FO3 - Removed unneeded Array Count aftersets

robertgk2017 committed:
FO3 - 2 More unneeded Array Count AfterSets

robertgk2017 committed:
Oops

robertgk2017 committed:
FO3/FNV \ IDLE - Update ANAM

robertgk2017 committed:
FO3 \ wbConditions - add wbFromVersion checks.

robertgk2017 committed:
FO3 \ wbConditions - tiny cleanup

robertgk2017 committed:
FNV - Update Conditions Handling

robertgk2017 committed:
FNV - Fix condition Parameter Order

robertgk2017 committed:
move wbAliasToStr to Common

robertgk2017 committed:
TES5 - Update Condition Handling

robertgk2017 committed:
TES4/FO3/FNV/TES5 \ Conditions formatting

robertgk2017 committed:
ALL - Update some Error Check messages

robertgk2017 committed:
TES5 - Forgot to sort the Paramater Numbers

robertgk2017 committed:
FO4 - Update Conditions Handling

robertgk2017 committed:
FO3/FNV/TES5/FO4/FO76/SF1 \ BPDT - Update BodyPart Array's SortKey and Summary.

robertgk2017 committed:
FO3/FNV \ NAVI - Decoded the rest of NVMI, Added Summaries/Sorting

robertgk2017 committed:
TES5/FO4/FO76/SF1 \ NAVI - Made more consistent across the games

robertgk2017 committed:
FO3/FNV \ Conditions - NVM that version check is 100% not valid.

robertgk2017 committed:
FNV \ TREE - Add 'Has Tree Lod' Flag.

robertgk2017 committed:
FNV \ WRLD \ PNAM - Set UnknownIsUnused on the flags

robertgk2017 committed:
TES3 - Misc organizational cleanup

robertgk2017 committed:
TES3 - Common Simple Defs

robertgk2017 committed:
FO3/FNV \ HDPT \ DATA - Fix Typo for Reachability

robertgk2017 committed:
TES5 \ DIAL \ BNAM - Make reachable

robertgk2017 committed:
wbImplementation \ TwbFile.BuildReachable - Update existing records' special handling to work across the games

robertgk2017 committed:
Assets Manager - Numerous Fixes

robertgk2017 committed:
Typos

robertgk2017 committed:
TES5 \ MSTT - Add Never Fades Flag

robertgk2017 committed:
ALL - Add INCC Updating to the save process and .SetRequired

robertgk2017 committed:
TES4/FO4 \ FLOR \ PFIG - Updated accepted form ids

robertgk2017 committed:
Assets Manager - TES5 \ Weather fixes

robertgk2017 committed:
This time for real

robertgk2017 committed:
Weather Editor Script - Updated to account for definition changes

jbostrus committed:
Scripts: report masters script now shows which masters were selecting during output

robertgk2017 committed:
Assets Manager - actually fixed FaceGen, for real this time.

robertgk2017 committed:
ALL \ IDLE \  DATA/ANAM - Update to reflect Parent/Previous IDLE Behavior

robertgk2017 committed:
ALL \ LAND \ VNML - Add GetCP to switch case cpIgnore vs cpNormal.

robertgk2017 committed:
Large Refs - Refs larger than size 32768 are not allowed to be Large Refs

robertgk2017 committed:
Assets Manager - TES5 \ HDPT - Add Parts

robertgk2017 committed:
ALL \ WRLD \ Worldspace Bounds - Update handling, also fixes OFST/Etc sub-record formatting in SF.

robertgk2017 committed:
FO3/FNV \ IDLE - Signature fix

robertgk2017 committed:
FO3/FNV \ EXPL \ DATA - Update Flags

robertgk2017 committed:
FNV \ WEAP \ VATS - Add aOptionalFromElement

robertgk2017 committed:
Common - tidy up gamemode IfThens

robertgk2017 committed:
ALL - wbEnchantment handling

robertgk2017 committed:
xeMainForm - Found a Typo

robertgk2017 committed:
TES5/FO4 \ wbConditions - Fix CIS1/CIS2 check

robertgk2017 committed:
ALL \ LAND - Update summaries

robertgk2017 committed:
And some typos

robertgk2017 committed:
And more

jbostrus committed:
Scripting: Add GetValue and GetElementValues to return resolved values such as decoded hashes.

robertgk2017 committed:
ALL \ SNDR \ Sounds - Collapse Sounds Array.

robertgk2017 committed:
BSArchPro - Add Support for script sources being in 'Source\Scripts'

robertgk2017 committed:
Assets Manager - Added support for Script Sources being in 'Source\Scripts' in SSE

robertgk2017 committed:
Assets Manager - Forgot a Backslash

robertgk2017 committed:
TES5/FO4/FO76/SF1 \ MATO - Update version checks/removed from Starfield.

robertgk2017 committed:
FO76 \ Conditions - Remove unused Paramter Types

robertgk2017 committed:
FO76 \ Conditions - rename CTDA to Conditions for consistency

robertgk2017 committed:
TES5/FO4 \ Conditions - Fix wbConditionStringToStr and wbConditionStringToInt logic

robertgk2017 committed:
ToStr instead of ToString

robertgk2017 committed:
FO76 \ Conditions - Move ParameterTypes and ConditionFunctions to top of Implementation Section, Collapse wbConditionParameters to 1 Array of IwbValueDef

robertgk2017 committed:
TES5/FO4 \ Conditions - Case checks in CIS1/CIS2 were redundant.

robertgk2017 committed:
FO4 \ Conditions - Condense wbConditionQuestOverlay

robertgk2017 committed:
ALL \ wbLandNormals - set default cp to Benign instead of Ignore

robertgk2017 committed:
ALL - Cleanup Common usage of wbQuestStageToInt

robertgk2017 committed:
FO76 \ Conditions - Cleanup/Condense/Organize Condition related functions

robertgk2017 committed:
ALL \ Conditions - More Formatting

robertgk2017 committed:
Large Refs - Ref size can go above 32768, needs more RE'ing

robertgk2017 committed:
ALL \ CPTH - Update ANAM Related Camera paths to properly represent the 2 links, and be consistent with other instances of this Struct

jbostrus committed:
Scripting/API: Allow adding masters with silent argument to prevent message log notification.

jbostrus committed:
Scripting/API: Add additional procedure AddMastersIfMissing to add multiple masters to a file with option to sort or silent add.

robertgk2017 committed:
FO76 \ Conditions - Sort ParamaterTypes

robertgk2017 committed:
SF1 \ Conditions - Remove Unused Parameter Types

robertgk2017 committed:
SF1 \ Conditions - Move Parameter Types and Functions to top of Defs File.

robertgk2017 committed:
ALL \ Conditions - Name all functions wbConditions.... instead of wbCTDA.... for consistency

robertgk2017 committed:
SF1 \ Conditions - Collapse Condition Parameters/Formatting cleanup

robertgk2017 committed:
SF1 \ Conditions - Move Used Functions to beginning of file after wbConditionFunctions

robertgk2017 committed:
SF1 \ CELL \ DATA - Add Unknown 9 Flag

eckserah committed:
Updates for fallout 76

Add XRLL which is reference lighting (same structure as XCLL which is cell lighting) and XRL2 which is reference lighting template (same as Cell LTMP which is cell lighting template) to REFR and all REFR records
Add FTAG Form Tags to GLOB, NPC_, PACK and RACE
Add PSCT, PSSC, PFDX, and PFDZ to PROJ
Add PBAP to BPTD
Add LVIT to LVLI
Add ACTV to EMOT (Currently unused)
Add PLYT Player Title record with subrecords of EDID, XALG, ANAM, BNAM, PTPR, PTSU, PTDS, and conditions.
Add Telegraph Data WTDT, WTDA, and WTDS to WEAP
Update header version to 224.

eckserah committed:
Updates for FO76

Updates to allow to work with the PTS as of Feb 2 2025. Version 0.2.1707.0
Add Condition functions for GetIsDisguisedWithKeyword and GetIsPlayerGhoul
Add entry points for Instant Reload Clip On Bash, Mod Attack Damage On Striking Appendage, Apply Spell On Actor When Limb Crippled, Mod Rads to Health Mult, Mod Rads to Radshield Mult, and Mod Weak Body Part Damage Mult
Add what seems to be entry check information for FACT. Has FCNR, FCRF, FCOR, FCBM, CITC, CTDA, and FCAF
Add FULL to TXST
Add DESC to EXPL
Add KNAM (Keyword) to OTFT
Add Unknown data at the end of QUST DATA
Add QUST QUCF
Add LOUT LMML
WEAP WSAM is Sneak Attack Multiplier

robertgk2017 committed:
TES5 \ FLOR \ SNAM - renamed to Harvest Sound to reflect the CK.

robertgk2017 committed:
FO76 - Merge Eck's PR to avoid github merge conflicts

robertgk2017 committed:
TES4 \ PGRD \ PGAG - Decoded by Kuro's PR, merged here to avoid github merge conflict

Kuroitsune commented:
Not to be a pedantic ass, but when you merged Eck's PR you fudged the header version a bit. It's 224, not 2224.

Just wanted to let you know before it got merged. :)

robertgk2017 committed:
OOPS

eckserah committed:
Updates for FO76

Add additional condition functions and weapon property types
Add record flags 2 and 9 to BOOK ADDN OTFT and PLYT
Add NULL as a valid record type for ALDN display name and RDES
Add AFOH to default objects

robertgk2017 committed:
Revert eck commit

jbostrus committed:
SF1: Fix INNR\XNAM\Target to detect INNR type and show contextual enum correctly

robertgk2017 committed:
Merge branch 'pr/1410' into Misc

robertgk2017 committed:
Merge branch 'pr/1400' into Misc

robertgk2017 committed:
Merge pull request #3 from jbostrus/SF1-INNR-Robert

SF1: Fix INNR\XNAM\Target to detect INNR type and show contextual enum correctly

robertgk2017 committed:
TES5 \ DOBJ \ DNAM - Decoded HMCC and HMAE

robertgk2017 committed:
TES5 \ MATO \ DATA \ Set OptionalFrom

robertgk2017 committed:
FO4 \ REFR \ XBSD - Update OptionalFrom tag
